packages/www/components/Register/index.tsx

```diff
@@ -88,66 +96,73 @@ const Login = ({ id, buttonText, onSubmit, loading, errors }) => {
             Create an account
           </Text>
 
-          <Text
-            variant="neutral"
-            size={1}
-            css={{
-              fontWeight: 600,
-              mb: "$1",
-              fontSize: "11px",
-              textTransform: "uppercase",
-            }}>
-            Account details
-          </Text>
-          <TextField
-            size="3"
-            id="firstName"
-            css={{
-              width: "100%",
-              mb: "$2",
-            }}
-            name="firstName"
-            type="text"
-            placeholder="First name"
-            required
-            value={firstName}
-            onChange={(e) => setFirstName(e.target.value)}
-          />
-          <TextField
-            size="3"
-            id="lastName"
-            css={{
-              width: "100%",
-              mb: "$2",
-            }}
-            name="lastName"
-            type="text"
-            placeholder="Last name"
-            required
-            value={lastName}
-            onChange={(e) => setLastName(e.target.value)}
-          />
-          <TextField
-            size="3"
-            id="email"
-            css={{
-              width: "100%",
-              mb: "$2",
-            }}
-            name="email"
-            type="email"
-            placeholder="Email"
-            required
-            value={email}
-            onChange={(e) => setEmail(e.target.value)}
-          />
+          <Flex
+            css={{ alignItems: "center", justifyContent: "center", mb: "$4" }}>
+            <Checkbox
+              onCheckedChange={(checked) =>
+                setIsCompanyAffiliated(checked === true)
+              }
+              checked={isCompanyAffiliated}
+            />
+            <Box css={{ ml: "$1" }}>I am using Livepeer Studio for work</Box>
+          </Flex>
+
+          {isCompanyAffiliated ? (
```

Comment:
Can we put this new field after the email/password? Feels kinda weird to be the first thing when it's some very optional information. Especially since we are defaulting this to `true` so it's always displayed there.

Would feel like a more natural flow to at least start with the standard registration fields like user/password and only then deal with auxiliary info/metadata like this.

Reply:
Yeah it changes the flow of the form, though, so I'm not sure about that. There are different fields depending on how you respond to this. cc @hthillman